test/integration/DataDrivenGremlinTest.py

```diff
@@ -16,9 +14,9 @@ class DataDrivenGremlinTest(IntegrationTest):
     def setUp(self):
         super().setUp()
 
-        self.client_provider = create_client_provider(self.auth_mode, self.iam_credentials_provider_type)
+        self.client = self.client_builder.build()
         query_check_for_airports = "g.V('3684').outE().inV().has(id, '3444')"
-        res = do_gremlin_query(query_check_for_airports, self.host, self.port, self.ssl, self.client_provider)
+        res = self.client.gremlin_query(query_check_for_airports)
```

Comment:
It would be better not to use explicit ID values here in case the data set ever changes and that route gets deleted. I am not sure what is needed but a different test might be more future proof.

Reply:
Yeah, this was put in place to ensure that all airports were added (by checking for the last one), we could instead rewrite it to look for the content.